cdc_rsync_cli/params.cc

```diff
@@ -348,6 +369,25 @@ bool CheckOptionResult(OptionResult result, const std::string& name,
   return true;
 }
 
+// Removes the user/host part of |destination| and puts it into |user_host|,
+// e.g. if |destination| is initially "user@foo.com:~/file", it is "~/file"
+// afterward and |user_host| is |user@foo.com|. Does not touch Windows drives,
+// e.g. C:\foo.
+void ParseUserHost(std::string* destination, std::string* user_host) {
```

Comment:
Reason is I don't need it. Both ssh and scp take user@host, so no need to split, but since the destination is handled within our tool, I need to split that off.